install-vmod/install-vmod

```diff
@@ -0,0 +1,77 @@
+#/bin/sh
+
+# usage:
+#     install_vmod URL [SHA512SUM]
+# deps:
+# - varnish development files
+# - curl
+# - pkg-config
+# - nproc
+# - make
+# - gcc/clang
+# - automake
+# - libtoolize
+# - whatever dependencies the vmod needs
+
+set -e
+
+check_bin() {
+	if ! command $@ &>/dev/null; then
```

Comment:
You should quote `"$@"` to prevent unintended word splitting, although current `check_bin` calls wouldn't be subject to it.